go/noise/laplace_noise_test.go

```diff
@@ -172,6 +172,525 @@ func TestDeltaForThresholdLaplace(t *testing.T) {
 		}
 	}
 }
+func TestInverseCDFLaplace(t *testing.T) {
+	for _, tc := range []struct {
+		desc                     string
+		mean, lambda, prob, want float64
+	}{
+		// 2 high-precision calculated random tests.
+		{
+			desc:   "High-precision random test",
+			mean:   64,
+			lambda: 4,
+			prob:   0.7875404240919761168041,
+			want:   67.4234254367},
+		{
+			desc:   "High-precision random test",
+			mean:   23,
+			lambda: 2,
+			prob:   0.1479685611330654517049,
+			want:   20.564783456,
+		},
+		// Edge cases where p = 0 or p = 1.(Output should be infinite)
```

Comment:
since we do not allow confidence levels of 1, it is irrelevant how inverseCDFLaplace behaves on this input. I don't think we should test for this, suggesting that this has some sort of meaning when it has not.

Reply:
It definitely doesn't have a meaning in terms of not allowing confidenceLevels to reach one or zero but it makes sense in terms of checking whether the function converges to infinities when you approach zero or one.

Reply:
Added low and high probabilities test for convergence.